packages/hint-compat-api/src/utils/ignore.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Determine the set of ignored features, given the provided defaults
+ * and user-specified `ignore` and/or `enable` options.
+ */
+export const resolveIgnore = (options: any, defaults: string[]) => {
+    const ignore = new Set(defaults);
+
+    if (options && options.ignore) {
+        options.ignore.forEach((entry: string) => {
```

Comment:
Not sure why I wrote this as a `forEach`. We've been preferring `for..of` loops lately.